toolchain/install/install_paths_test_helpers.h

```diff
@@ -0,0 +1,19 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_INSTALL_INSTALL_PATHS_TEST_HELPERS_H_
+#define CARBON_TOOLCHAIN_INSTALL_INSTALL_PATHS_TEST_HELPERS_H_
+
+#include "llvm/Support/VirtualFileSystem.h"
+#include "toolchain/install/install_paths.h"
+
+namespace Carbon::Testing {
+
+// Prepares the VFS with prelude files from the real filesystem.
+auto CopyPreludeFiles(InstallPaths install_paths,
+                      llvm::vfs::InMemoryFileSystem* vfs) -> void;
```

Comment:
For me it'd be clearer to name this `AddPreludeFiles` or `AddPreludeFilesToVfs` or something. This name sounds superficially like it would perform an actual on-disk file copy.

Reply:
How about `CopyPreludeFilesToVfs`?

Reply:
Sure, that's clear enough that it's not touching the disk.

Reply:
Switched to `AddPreludeFilesToVfs`